library/ZendService/Apple/Apns/Message.php

```diff
@@ -256,6 +262,32 @@ public function setSound($sound)
     }
 
     /**
+     * Get Mutable Content
+     * 
+     * @return int|null
+     */
+    public function getMutableContent()
+    {
+        return $this->mutableContent;
+    }
+
+    /**
+     * Set Mutable Content
+     * 
+     * @param int|null $value
```

Comment:
Is any integer acceptable? What about negative values? Is there documentation for this?